db/seeds.rb

```diff
@@ -63,11 +178,33 @@
                       price: 300,
                       brand: 'Ride',
                       image: File.open('app/assets/images/bidndings1.jpg')
-              },
+                    },
 
                     { name: 'Anti-kickflippers',
                       description: 'No kickflips allow with the Salomon Anti-kickflippers',
                       price: 150,
                       brand: 'Salomon',
                       image: File.open('app/assets/images/bindings2.jpg')
+                    },
+                    { name: 'EX BINDINGS',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.